src/Icon/Icon.js

```diff
@@ -8,12 +9,14 @@ import cx from 'classnames';
   icon: t.maybe(t.Str),
   className: t.maybe(t.Str),
   style: t.maybe(t.Obj),
+  paths: t.maybe(t.refinements(t.Number, x => x - Math.floor(x) === 0 && x > 0)),
```

Comment:
please define `const Integer = t.refinement(t.Number, (n) => n % 1 === 0, 'Integer');` (copy-pasted from `tcomb` examples)
and then `paths: t.maybe(Integer)`